### PR TITLE
21.1.5 update

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
-
-
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4867)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+
+# WPF Dock Layout Manager - Generate a Dock UI with the MVVM Design Pattern
+
+The following creates [LayoutPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutPanel) and [DocumentPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel) objects from a data source according to the MVVM design pattern and implements template selection.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
The following example creates [LayoutPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutPanel) and [DocumentPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel) objects from a data source according to the MVVM design pattern and implements template selection.
```

File: Readme.md [hunk 2]
```diff
@@ -1,22 +1,35 @@
...
+
+<img src="https://user-images.githubusercontent.com/12169834/175355745-d50cc84d-c36e-4f4b-9db5-7b0dbd211b8a.png" width=600px/>
+
+In this example, the **MainViewModel** class contains the **Items** collection that contains objects that are displayed as layout items in the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager). An instance of the **MainViewModel** class is assigned to the Window's **DataContext**. The [DockLayoutManager.ItemsSource](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager.ItemsSource) property is bound to the **Items** property in the **MainViewModel**. In this case, the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) gets the source of objects to  visualize it as layout items.
```
Comment (albertov05) [inline Readme.md]: `In this case, the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) gets the source of objects to  visualize it as layout items.`

What is "it"?

File: Readme.md [hunk 3]
```diff
@@ -1,22 +1,35 @@
...
+
+The **Items** collection contains the following types of items:
+
+1. **ViewModel** objects - contain data to create [LayoutPanels](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutPanel)
```
Comment (albertov05) [inline Readme.md]:
```suggestion
1. **ViewModel** objects — contain data to create [LayoutPanels](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutPanel)
```

File: Readme.md [hunk 4]
```diff
@@ -1,22 +1,35 @@
...
+
+1. **DocumentViewModel** objects - contain data to create [DocumentPanels](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel)
```
Comment (albertov05) [inline Readme.md]:
```suggestion
1. **DocumentViewModel** objects — contain data to create [DocumentPanels](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel)
```

File: Readme.md (hunk 2 above)
Comment (friedcucumber) [reply Readme.md]:
```suggestion
In this example, the **MainViewModel** class contains the **Items** collection that contains objects that are displayed as layout items in the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager). An instance of the **MainViewModel** class is assigned to the Window's **DataContext**. The [DockLayoutManager.ItemsSource](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager.ItemsSource) property is bound to the **Items** property in the **MainViewModel**. In this case, the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) gets the source of objects to visualize them as layout items.

```

**Pull request merged**